spec/views/tutors/show.html.erb_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'tutors/show.html.erb', type: :view do
+  xdescribe 'populate page and links' do
+    before do
+      @tutor = Tutor.create(
+        first_name: 'Viviano', last_name: 'Cantu',
+        email_preferred: 'email@email.com',
+        address1: '200 Huntington St', city: 'Boston', state: 'MA',
+        zip: '02120', home_phone: '1112223333', cell_phone: '4445556666',
+        affiliate: 'Affiliate1', gender: 'male', native_language: 'English',
+        race: 'white'
+      )
+      @students = [Student.create(first_name: 'Mike', last_name: 'White'),
+                   Student.create(first_name: 'Alex', last_name: 'Jones')]
+      render
+    end
+
+    describe 'presence of tutors' do
+      it 'presence of name' do
```

Comment:
`it presence of name` doesn't make any sense.  The reason it's worded like that is because in the spec output that's how it reads. maybe `it shows the name` or something.